qdp/Makefile

```diff
@@ -43,7 +43,7 @@ test_rust:
 	cargo test --workspace
 
 install_benchmark:
-	cd qdp/qdp-python && uv sync --group benchmark
+	cd qdp-python && uv sync --group benchmark
```

Comment:
Looks like terminal is in different directory! Could we use `cd -` to make it at same starting point first?